docs/reference/dependencies.md

```diff
@@ -2898,4 +2899,466 @@ Connectors dependencies are packaged with the [release assets](https://github.co
+- [set-function-length](https://github.com/ljharb/set-function-length#readme) (MIT)
+- [shallow-equal](https://github.com/moroshko/shallow-equal#readme) (MIT)
+- [shebang-command](https://github.com/kevva/shebang-command#readme) (MIT)
+- [shebang-regex](https://github.com/sindresorhus/shebang-regex#readme) (MIT)
+- [side-channel](https://github.com/ljharb/side-channel#readme) (MIT)
+- [statuses](https://github.com/jshttp/statuses#readme) (MIT)
+- [string-width](https://github.com/sindresorhus/string-width#readme) (MIT)
+- [strip-ansi](https://github.com/chalk/strip-ansi#readme) (MIT)
+- [supports-color](https://github.com/chalk/supports-color#readme) (MIT)
+- [swagger-ui-express](https://github.com/scottie1984/swagger-ui-express) (MIT)
+- [tdigest](https://github.com/welch/tdigest) (MIT)
+- [toidentifier](https://github.com/component/toidentifier#readme) (MIT)
+- [ts-essentials](https://github.com/krzkaczor/ts-essentials#readme) (MIT)
+- [tsoa](https://github.com/lukeautry/tsoa#readme) (MIT)
+- [type-is](https://github.com/jshttp/type-is#readme) (MIT)
+- [undici-types](https://undici.nodejs.org) (MIT)
+- [universalify](https://github.com/RyanZim/universalify#readme) (MIT)
+- [unpipe](https://github.com/stream-utils/unpipe#readme) (MIT)
+- [utils-merge](https://github.com/jaredhanson/utils-merge#readme) (MIT)
+- [validator](https://github.com/validatorjs/validator.js) (MIT)
+- [vary](https://github.com/jshttp/vary#readme) (MIT)
+- [wordwrap](https://github.com/substack/node-wordwrap#readme) (MIT)
+- [wrap-ansi](https://github.com/chalk/wrap-ansi#readme) (MIT)
+- [yamljs](https://github.com/jeremyfa/yaml.js#readme) (MIT)
+- [yargs](https://yargs.js.org/) (MIT)
+- [zod](https://zod.dev) (MIT)
+- [buffer-equal-constant-time](https://github.com/goinstant/buffer-equal-constant-time#readme) (BSD-3-Clause)
+- [qs](https://github.com/ljharb/qs) (BSD-3-Clause)
+- [source-map](https://github.com/mozilla/source-map) (BSD-3-Clause)
+- [sprintf-js](https://github.com/alexei/sprintf.js#readme) (BSD-3-Clause)
+- [dotenv](https://github.com/motdotla/dotenv#readme) (BSD-2-Clause)
+- [ecdsa-sig-formatter](https://github.com/Brightspace/node-ecdsa-sig-formatter#readme) (Apache-2.0)
+- [prom-client](https://github.com/siimon/prom-client) (Apache-2.0)
+- [reflect-metadata](http://rbuckton.github.io/reflect-metadata) (Apache-2.0)
+- [swagger-ui-dist](https://github.com/swagger-api/swagger-ui#readme) (Apache-2.0)
+- [typescript](https://www.typescriptlang.org/) (Apache-2.0)
+- [jackspeak](https://github.com/isaacs/jackspeak#readme) (BlueOak-1.0.0)
+- [path-scurry](https://github.com/isaacs/path-scurry#readme) (BlueOak-1.0.0)
+- [tslib](https://www.typescriptlang.org/) (0BSD)
+- [url-value-parser](https://github.com/disjunction/url-value-parser#readme) (MIT-0)
+
+### Console Self-Managed Dependencies (frontend)
+
+- [@ampproject/remapping](https://github.com/ampproject/remapping#readme) (Apache-2.0)
+- [@camunda/camunda-composite-components](undefined) (Apache-2.0)
```

Comment:
Need to add a link here -- closest I could locate was https://www.npmjs.com/package/@camunda/camunda-optimize-composite-components.